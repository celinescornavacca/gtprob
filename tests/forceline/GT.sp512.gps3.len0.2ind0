(((((((((YH1,YH2),YH0),((ZH1,ZH2),ZH0)),(((((((MD0,(MD1,MD2)),(((OI1,OI2),OI0),(((VI1,VI2),VI0),((WI1,WI2),WI0)))),(((((DE1,DE2),DE0),((DT0,(DT1,DT2)),(((IT1,IT2),IT0),((JT1,JT2),JT0)))),((((CN0,(CN1,CN2)),(DN0,(DN1,DN2))),((NJ0,(NJ1,NJ2)),(OJ0,(OJ1,OJ2)))),((CB1,CB2),CB0))),(((KC1,KC2),KC0),(LC0,(LC1,LC2))))),(((ID1,ID2),ID0),((JD1,JD2),JD0))),((((IC0,(IC1,IC2)),(JC0,(JC1,JC2))),(((PA1,PA2),PA0),(((((XG1,XG2),XG0),(YG0,(YG1,YG2))),(((YP1,YP2),YP0),(ZP0,(ZP1,ZP2)))),((XR0,(XR1,XR2)),(YR0,(YR1,YR2)))))),((((IQ1,IQ2),IQ0),(JQ0,(JQ1,JQ2))),(EG0,(EG1,EG2))))),(((TP0,(TP1,TP2)),(UP0,(UP1,UP2))),((SC0,(SC1,SC2)),(TC0,(TC1,TC2))))),((((AB1,AB2),AB0),((((HS0,(HS1,HS2)),(IS0,(IS1,IS2))),((YD0,(YD1,YD2)),(ZD0,(ZD1,ZD2)))),((((QG1,QG2),QG0),((RG1,RG2),RG0)),((((FO1,FO2),FO0),((GO1,GO2),GO0)),(((ER1,ER2),ER0),((FR1,FR2),FR0)))))),((((LJ0,(LJ1,LJ2)),(MJ0,(MJ1,MJ2))),(KI0,(KI1,KI2))),((UA1,UA2),UA0))))),(((((((GM1,GM2),GM0),((HM1,HM2),HM0)),((SE1,SE2),SE0)),((((EH0,(EH1,EH2)),(((VN1,VN2),VN0),(WN0,(WN1,WN2)))),((((((FE1,FE2),FE0),(GE0,(GE1,GE2))),((((VM1,VM2),VM0),(((JP1,JP2),JP0),(KP0,(KP1,KP2)))),(VF0,(VF1,VF2)))),(VB0,(VB1,VB2))),((((((PR1,PR2),PR0),((QR1,QR2),QR0)),(((EN1,EN2),EN0),((XO0,(XO1,XO2)),(YO0,(YO1,YO2))))),(JL0,(JL1,JL2))),(NF0,(NF1,NF2))))),((((((DC1,DC2),DC0),(((AD1,AD2),AD0),((BD1,BD2),BD0))),(ZB0,(ZB1,ZB2))),((((TN0,(TN1,TN2)),(UN0,(UN1,UN2))),((UQ0,(UQ1,UQ2)),((VQ1,VQ2),VQ0))),(((((CR1,CR2),CR0),(DR0,(DR1,DR2))),((RP0,(RP1,RP2)),((SP1,SP2),SP0))),(NE0,(NE1,NE2))))),(((RA1,RA2),RA0),((((CD1,CD2),CD0),(((UD1,UD2),UD0),((EF0,(EF1,EF2)),(((QL1,QL2),QL0),(RL0,(RL1,RL2)))))),(((WG0,(WG1,WG2)),(((HI1,HI2),HI0),(II0,(II1,II2)))),((MM0,(MM1,MM2)),((NM1,NM2),NM0)))))))),(((((((((SL1,SL2),SL0),((TL1,TL2),TL0)),((KL1,KL2),KL0)),((HJ0,(HJ1,HJ2)),(IJ0,(IJ1,IJ2)))),(((IH0,(IH1,IH2)),(JH0,(JH1,JH2))),((((ZN1,ZN2),ZN0),(AO0,(AO1,AO2))),(SM0,(SM1,SM2))))),((HA0,(HA1,HA2)),((XC0,(XC1,XC2)),((MI0,(MI1,MI2)),(NI0,(NI1,NI2)))))),((((VG1,VG2),VG0),(((QO1,QO2),QO0),((RO1,RO2),RO0))),((((((UF1,UF2),UF0),((WK0,(WK1,WK2)),(XK0,(XK1,XK2)))),(((CH1,CH2),CH0),((DH1,DH2),DH0))),(((((ZL0,(ZL1,ZL2)),(((GT1,GT2),GT0),((HT1,HT2),HT0))),((XF1,XF2),XF0)),((FL0,(FL1,FL2)),(GL0,(GL1,GL2)))),(((UH0,(UH1,UH2)),(((KT1,KT2),KT0),((LT1,LT2),LT0))),(((VR1,VR2),VR0),((WR1,WR2),WR0))))),(((TH0,(TH1,TH2)),(((JK1,JK2),JK0),((KK1,KK2),KK0))),((ZC1,ZC2),ZC0))))),(((EM0,(EM1,EM2)),(FM0,(FM1,FM2))),((((XP1,XP2),XP0),((((QT1,QT2),QT0),((RT1,RT2),RT0)),((JS0,(JS1,JS2)),(KS0,(KS1,KS2))))),((AE1,AE2),AE0))))),((((NA1,NA2),NA0),((OA1,OA2),OA0)),(((JA1,JA2),JA0),(((((LN1,LN2),LN0),(((TS1,TS2),TS0),(US0,(US1,US2)))),(IE0,(IE1,IE2))),((((FP0,(FP1,FP2)),(GP0,(GP1,GP2))),((((VK1,VK2),VK0),(((RR0,(RR1,RR2)),(SR0,(SR1,SR2))),(UL0,(UL1,UL2)))),((LI1,LI2),LI0))),(LE0,(LE1,LE2)))))))),((((UC1,UC2),UC0),((((JO0,(JO1,JO2)),((KO1,KO2),KO0)),(SH0,(SH1,SH2))),((QD0,(QD1,QD2)),((RD1,RD2),RD0)))),((((OT0,(OT1,OT2)),(PT0,(PT1,PT2))),((PP1,PP2),PP0)),((QB1,QB2),QB0)))),(((((TF0,(TF1,TF2)),(((MT1,MT2),MT0),(NT0,(NT1,NT2)))),((ZE0,(ZE1,ZE2)),(AF0,(AF1,AF2)))),(((UB0,(UB1,UB2)),(((((NO1,NO2),NO0),((XS0,(XS1,XS2)),(YS0,(YS1,YS2)))),((BO0,(BO1,BO2)),(CO0,(CO1,CO2)))),((PF0,(PF1,PF2)),(((MR1,MR2),MR0),((NS0,(NS1,NS2)),((OS1,OS2),OS0)))))),((VO0,(VO1,VO2)),((WO1,WO2),WO0)))),(((((CC0,(CC1,CC2)),((BJ0,(BJ1,BJ2)),((CJ1,CJ2),CJ0))),((KA1,KA2),KA0)),((((((MF1,MF2),MF0),(((CI1,CI2),CI0),((DI1,DI2),DI0))),((((IM1,IM2),IM0),((JM1,JM2),JM0)),((VJ1,VJ2),VJ0))),((((DA1,DA2),DA0),((EA1,EA2),EA0)),(((FC0,(FC1,FC2)),(((HK1,HK2),HK0),(IK0,(IK1,IK2)))),(((FB1,FB2),FB0),((RE0,(RE1,RE2)),(((LK1,LK2),LK0),((MK1,MK2),MK0))))))),((((WA1,WA2),WA0),(((KR0,(KR1,KR2)),((LR1,LR2),LR0)),(QE0,(QE1,QE2)))),(((GC0,(GC1,GC2)),((GJ0,(GJ1,GJ2)),((((QP1,QP2),QP0),(((SQ1,SQ2),SQ0),((TQ1,TQ2),TQ0))),(UO0,(UO1,UO2))))),(((IG1,IG2),IG0),(JG0,(JG1,JG2))))))),(((((((FN0,(FN1,FN2)),(GN0,(GN1,GN2))),(((GR1,GR2),GR0),(HR0,(HR1,HR2)))),((TE1,TE2),TE0)),((OB0,(OB1,OB2)),((PB1,PB2),PB0))),(BA0,(BA1,BA2))),((((((MH1,MH2),MH0),((NH1,NH2),NH0)),((((PQ1,PQ2),PQ0),(QQ0,(QQ1,QQ2))),(((VE1,VE2),VE0),(WE0,(WE1,WE2))))),(((RB1,RB2),RB0),(SB0,(SB1,SB2)))),((IA1,IA2),IA0)))))),((((((JN1,JN2),JN0),(KN0,(KN1,KN2))),(((OL1,OL2),OL0),((PL1,PL2),PL0))),((((BL1,BL2),BL0),(CL0,(CL1,CL2))),(VD0,(VD1,VD2)))),(((((((OH0,(OH1,OH2)),((PH1,PH2),PH0)),((JF1,JF2),JF0)),(XE0,(XE1,XE2))),(((HL1,HL2),HL0),((IL1,IL2),IL0))),(((BB0,(BB1,BB2)),(((((NQ1,NQ2),NQ0),((OQ1,OQ2),OQ0)),((JJ0,(JJ1,JJ2)),(KJ0,(KJ1,KJ2)))),((((FG1,FG2),FG0),((AN0,(AN1,AN2)),((BN1,BN2),BN0))),((((EQ1,EQ2),EQ0),((FQ1,FQ2),FQ0)),((((QM1,QM2),QM0),(((HO1,HO2),HO0),(IO0,(IO1,IO2)))),((FF1,FF2),FF0)))))),((BE0,(BE1,BE2)),(CE0,(CE1,CE2))))),((((((XB1,XB2),XB0),((YB1,YB2),YB0)),(WB0,(WB1,WB2))),(((((OG1,OG2),OG0),(PG0,(PG1,PG2))),(((SG0,(SG1,SG2)),((TG1,TG2),TG0)),((GB1,GB2),GB0))),((((((QK1,QK2),QK0),((RK0,(RK1,RK2)),((SK1,SK2),SK0))),(((((KQ1,KQ2),KQ0),((LQ1,LQ2),LQ0)),((TB1,TB2),TB0)),((QF0,(QF1,QF2)),(RF0,(RF1,RF2))))),((((EC1,EC2),EC0),((PN0,(PN1,PN2)),(QN0,(QN1,QN2)))),(((SJ1,SJ2),SJ0),(TJ0,(TJ1,TJ2))))),(((((AR1,AR2),AR0),(BR0,(BR1,BR2))),((((VL1,VL2),VL0),(WL0,(WL1,WL2))),((DD0,(DD1,DD2)),(ED0,(ED1,ED2))))),((QA1,QA2),QA0))))),(((((HN0,(HN1,HN2)),((IN1,IN2),IN0)),(((DP1,DP2),DP0),((EP1,EP2),EP0))),((((LB1,LB2),LB0),((MB1,MB2),MB0)),((((VC1,VC2),VC0),(WC0,(WC1,WC2))),(((((WQ1,WQ2),WQ0),(XQ0,(XQ1,XQ2))),((FS0,(FS1,FS2)),(GS0,(GS1,GS2)))),(((CK1,CK2),CK0),(((AM1,AM2),AM0),(BM0,(BM1,BM2)))))))),((XA0,(XA1,XA2)),(((XI1,XI2),XI0),((YI1,YI2),YI0)))))))),((((((UE0,(UE1,UE2)),((DJ0,(DJ1,DJ2)),(((NN1,NN2),NN0),(ON0,(ON1,ON2))))),(((((((RC1,RC2),RC0),(((GD1,GD2),GD0),(HD0,(HD1,HD2)))),(((PE1,PE2),PE0),(((WJ1,WJ2),WJ0),((XJ1,XJ2),XJ0)))),((((RJ1,RJ2),RJ0),((ET0,(ET1,ET2)),((FT1,FT2),FT0))),(DB0,(DB1,DB2)))),(((WM0,(WM1,WM2)),((XM1,XM2),XM0)),((MC1,MC2),MC0))),(((TK0,(TK1,TK2)),((UK1,UK2),UK0)),((NB1,NB2),NB0)))),((((((NR1,NR2),NR0),(OR0,(OR1,OR2))),(((((OE1,OE2),OE0),((LO0,(LO1,LO2)),((MO1,MO2),MO0))),(((KM0,(KM1,KM2)),((LM1,LM2),LM0)),(LL0,(LL1,LL2)))),((((KG1,KG2),KG0),((LG1,LG2),LG0)),(FD0,(FD1,FD2))))),((((((KD1,KD2),KD0),(LD0,(LD1,LD2))),(((SO1,SO2),SO0),((TO1,TO2),TO0))),(((ZI0,(ZI1,ZI2)),((AJ1,AJ2),AJ0)),(UG0,(UG1,UG2)))),((((KF1,KF2),KF0),(LF0,(LF1,LF2))),((((FK0,(FK1,FK2)),(GK0,(GK1,GK2))),((OM0,(OM1,OM2)),((PM1,PM2),PM0))),(((SD1,SD2),SD0),(TD0,(TD1,TD2))))))),(((((PK1,PK2),PK0),(((VP1,VP2),VP0),(WP0,(WP1,WP2)))),(HH0,(HH1,HH2))),(((AK1,AK2),AK0),((BK1,BK2),BK0))))),((((((((((BH0,(BH1,BH2)),(((TM1,TM2),TM0),(UM0,(UM1,UM2)))),((QC1,QC2),QC0)),(((NG0,(NG1,NG2)),(((RH1,RH2),RH0),((RN0,(RN1,RN2)),(SN0,(SN1,SN2))))),(((HE1,HE2),HE0),(((MQ1,MQ2),MQ0),((IR0,(IR1,IR2)),(JR0,(JR1,JR2))))))),(((KB1,KB2),KB0),((HP0,(HP1,HP2)),(IP0,(IP1,IP2))))),((MA0,(MA1,MA2)),((AC0,(AC1,AC2)),((BC1,BC2),BC0)))),(((CM1,CM2),CM0),(DM0,(DM1,DM2)))),((AA1,AA2),AA0)),(((((((KE1,KE2),KE0),((WH0,(WH1,WH2)),((XH1,XH2),XH0))),((((((TR1,TR2),TR0),((UR1,UR2),UR0)),(ND0,(ND1,ND2))),(((HF1,HF2),HF0),(IF0,(IF1,IF2)))),((RS0,(RS1,RS2)),((SS1,SS2),SS0)))),(((((((RI1,RI2),RI0),(SI0,(SI1,SI2))),((RQ0,(RQ1,RQ2)),((LS0,(LS1,LS2)),(MS0,(MS1,MS2))))),(((((EJ1,EJ2),EJ0),((FJ1,FJ2),FJ0)),(((PS1,PS2),PS0),(QS0,(QS1,QS2)))),(((WD1,WD2),WD0),(XD0,(XD1,XD2))))),((((VH0,(VH1,VH2)),((ZK0,(ZK1,ZK2)),(AL0,(AL1,AL2)))),(((DK0,(DK1,DK2)),((EK1,EK2),EK0)),((AQ0,(AQ1,AQ2)),((BQ1,BQ2),BQ0)))),(((OO0,(OO1,OO2)),(PO0,(PO1,PO2))),((WF1,WF2),WF0)))),(((((NK1,NK2),NK0),((ZR0,(ZR1,ZR2)),(AS0,(AS1,AS2)))),(OF0,(OF1,OF2))),((GI0,(GI1,GI2)),(((NP1,NP2),NP0),(OP0,(OP1,OP2))))))),((FA1,FA2),FA0)),((((JB1,JB2),JB0),((((SF1,SF2),SF0),(((LP1,LP2),LP0),((MP1,MP2),MP0))),((AG0,(AG1,AG2)),((BG1,BG2),BG0)))),(((FH0,(FH1,FH2)),((GH1,GH2),GH0)),(((HC1,HC2),HC0),(((NC1,NC2),NC0),((YC0,(YC1,YC2)),((ZO0,(ZO1,ZO2)),(AP0,(AP1,AP2)))))))))),(((((YA1,YA2),YA0),((((PJ1,PJ2),PJ0),(QJ0,(QJ1,QJ2))),(((YK1,YK2),YK0),((ML0,(ML1,ML2)),((NL1,NL2),NL0))))),((((GA1,GA2),GA0),((((((ZS1,ZS2),ZS0),((AT1,AT2),AT0)),(((BF0,(BF1,BF2)),((CF1,CF2),CF0)),((JE1,JE2),JE0))),((((MG0,(MG1,MG2)),(((AI1,AI2),AI0),((BI1,BI2),BI0))),(PD0,(PD1,PD2))),((EI0,(EI1,EI2)),((FI1,FI2),FI0)))),((((QH1,QH2),QH0),((OK0,(OK1,OK2)),((((GQ1,GQ2),GQ0),(HQ0,(HQ1,HQ2))),((MN1,MN2),MN0)))),(EB0,(EB1,EB2))))),((CA1,CA2),CA0))),(((((((YF1,YF2),YF0),((ZF1,ZF2),ZF0)),(EE0,(EE1,EE2))),((((PI1,PI2),PI0),((QI1,QI2),QI0)),((ME1,ME2),ME0))),(((CG0,(CG1,CG2)),(DG0,(DG1,DG2))),(((XN1,XN2),XN0),((YN1,YN2),YN0)))),(((LA1,LA2),LA0),((GG0,(GG1,GG2)),((HG1,HG2),HG0))))))),(((((((((KH1,KH2),KH0),(LH0,(LH1,LH2))),((IB1,IB2),IB0)),((((TI0,(TI1,TI2)),(UI0,(UI1,UI2))),(GF0,(GF1,GF2))),((((BS0,(BS1,BS2)),((CS1,CS2),CS0)),((YL1,YL2),YL0)),((ZG0,(ZG1,ZG2)),((AH1,AH2),AH0))))),(ZA0,(ZA1,ZA2))),(SA0,(SA1,SA2))),((VA0,(VA1,VA2)),(((((BT1,BT2),BT0),((CT1,CT2),CT0)),((OC0,(OC1,OC2)),((PC1,PC2),PC0))),((((YM0,(YM1,YM2)),((ZM1,ZM2),ZM0)),((RM0,(RM1,RM2)),((BP0,(BP1,BP2)),((CP1,CP2),CP0)))),(((((DL1,DL2),DL0),(EL0,(EL1,EL2))),(((JI1,JI2),JI0),((UJ0,(UJ1,UJ2)),((DO0,(DO1,DO2)),(EO0,(EO1,EO2)))))),(DF0,(DF1,DF2))))))),((((((YJ0,(YJ1,YJ2)),((ZJ1,ZJ2),ZJ0)),((YE1,YE2),YE0)),((HB1,HB2),HB0)),((((OD1,OD2),OD0),((YQ0,(YQ1,YQ2)),(ZQ0,(ZQ1,ZQ2)))),((DS0,(DS1,DS2)),(ES0,(ES1,ES2))))),(((TA1,TA2),TA0),(((XL0,(XL1,XL2)),(((CQ1,CQ2),CQ0),(DQ0,(DQ1,DQ2)))),(((VS1,VS2),VS0),(WS0,(WS1,WS2)))))))));
