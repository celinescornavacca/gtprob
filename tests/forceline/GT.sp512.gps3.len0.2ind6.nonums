((((((((((JJ,(JJ,JJ)),((KJ,KJ),KJ)),((NE,NE),NE)),(((((WF,WF),WF),((ZJ,(ZJ,ZJ)),(AK,(AK,AK)))),(((FD,FD),FD),(((HF,HF),HF),((IF,IF),IF)))),((((((AE,AE),AE),(((DE,DE),DE),((DG,(DG,DG)),(EG,(EG,EG))))),((((JG,(JG,JG)),(((((IQ,(IQ,IQ)),(JQ,(JQ,JQ))),((ZO,ZO),ZO)),((LN,(LN,LN)),(MN,(MN,MN)))),(XL,(XL,XL)))),((KF,(KF,KF)),((LF,LF),LF))),((((VG,VG),VG),(((OO,OO),OO),((PO,PO),PO))),(DF,(DF,DF))))),((VA,VA),VA)),(UA,(UA,UA))))),(((QI,QI),QI),(RI,(RI,RI)))),(((((QD,QD),QD),((RD,RD),RD)),(WB,(WB,WB))),(((((FC,FC),FC),(((((XH,XH),XH),((WS,(WS,WS)),(XS,(XS,XS)))),(((RM,RM),RM),(((YS,YS),YS),(ZS,(ZS,ZS))))),((PK,(PK,PK)),((MP,(MP,MP)),(NP,(NP,NP)))))),(EB,(EB,EB))),(((CS,CS),CS),((DS,DS),DS))))),((((SH,SH),SH),(TH,(TH,TH))),(((((CC,CC),CC),(((II,(II,II)),((((ET,ET),ET),(FT,(FT,FT))),(QP,(QP,QP)))),(ID,(ID,ID)))),((GD,(GD,GD)),(((UD,UD),UD),((UL,(UL,UL)),(VL,(VL,VL)))))),((FB,(FB,FB)),((GB,GB),GB))))),(((((FF,(FF,FF)),(GF,(GF,GF))),((RP,(RP,RP)),((SP,SP),SP))),((IA,IA),IA)),(((((CB,(CB,CB)),(((ZK,(ZK,ZK)),((AL,AL),AL)),((VP,(VP,VP)),((WP,WP),WP)))),(((((((SM,SM),SM),((TM,TM),TM)),((NL,(NL,NL)),(OL,(OL,OL)))),((XG,XG),XG)),((((WN,WN),WN),(XN,(XN,XN))),((CI,(CI,CI)),((DI,DI),DI)))),((((MK,MK),MK),(NK,(NK,NK))),(((((SL,SL),SL),(TL,(TL,TL))),((MF,MF),MF)),(((PL,(PL,PL)),(((XR,(XR,XR)),((YR,YR),YR)),(XM,(XM,XM)))),((OG,OG),OG)))))),((((QB,(QB,QB)),((IG,(IG,IG)),(((FN,FN),FN),(GN,(GN,GN))))),((WA,WA),WA)),((KB,(KB,KB)),((RG,(RG,RG)),((VQ,(VQ,VQ)),(WQ,(WQ,WQ))))))),((((YH,YH),YH),(((FP,(FP,FP)),(GP,(GP,GP))),((BI,BI),BI))),(((GE,(GE,GE)),((((UQ,UQ),UQ),((IS,(IS,IS)),((JS,JS),JS))),((GK,GK),GK))),(((XO,(XO,XO)),((((KT,KT),KT),(LT,(LT,LT))),((EP,EP),EP))),(((KS,KS),KS),(LS,(LS,LS))))))))),(((((OE,OE),OE),((((UJ,UJ),UJ),((VJ,VJ),VJ)),((OH,OH),OH))),((CJ,(CJ,CJ)),((VR,(VR,VR)),(WR,(WR,WR))))),(((((HJ,(HJ,HJ)),((RN,(RN,RN)),((SN,SN),SN))),((OI,OI),OI)),((GG,(GG,GG)),(HG,(HG,HG)))),(((FQ,FQ),FQ),((GR,(GR,GR)),((HR,HR),HR)))))),((((((((((MT,(MT,MT)),((NT,NT),NT)),((LJ,(LJ,LJ)),(((IT,IT),IT),(JT,(JT,JT))))),((((QC,QC),QC),((SD,(SD,SD)),((XI,(XI,XI)),(YI,(YI,YI))))),((((SK,(SK,SK)),(((LL,LL),LL),(ML,(ML,ML)))),((SC,SC),SC)),((IL,(IL,IL)),(((GM,GM),GM),(HM,(HM,HM))))))),((((LC,(LC,LC)),(MC,(MC,MC))),((RR,(RR,RR)),((SR,SR),SR))),(LA,(LA,LA)))),(((((XD,XD),XD),(((RE,RE),RE),((SE,SE),SE))),(((MS,MS),MS),(NS,(NS,NS)))),(((WG,WG),WG),(((SS,(SS,SS)),((TS,TS),TS)),((BQ,BQ),BQ))))),((((VK,VK),VK),(WK,(WK,WK))),(((((RF,(RF,RF)),((((FR,FR),FR),((PR,(PR,PR)),((QR,QR),QR))),((SI,SI),SI))),((UB,UB),UB)),(((((((VE,VE),VE),((KR,(KR,KR)),((LR,LR),LR))),((TA,TA),TA)),(((((((OS,OS),OS),(PS,(PS,PS))),((YK,YK),YK)),((PJ,(PJ,PJ)),(QJ,(QJ,QJ)))),((OC,OC),OC)),(((SB,SB),SB),((WJ,(WJ,WJ)),(((DM,DM),DM),(((ZQ,ZQ),ZQ),(AR,(AR,AR)))))))),((NA,(NA,NA)),((((((ZG,(ZG,ZG)),((QK,(QK,QK)),((RK,RK),RK))),((PC,PC),PC)),((NC,NC),NC)),(((JP,(JP,JP)),((XP,(XP,XP)),(YP,(YP,YP)))),(((VC,VC),VC),((WC,WC),WC)))),(((IC,IC),IC),(JC,(JC,JC)))))),((((BN,BN),BN),((CN,CN),CN)),(RH,(RH,RH))))),(((YB,YB),YB),(ZB,(ZB,ZB)))))),(((((FA,(FA,FA)),((XB,(XB,XB)),(((JH,JH),JH),((KH,KH),KH)))),(((((((AT,AT),AT),((BT,BT),BT)),((PQ,PQ),PQ)),(((OB,OB),OB),((PB,(PB,PB)),(((XC,XC),XC),(((OT,OT),OT),((PT,PT),PT)))))),(((SQ,(SQ,SQ)),((TQ,TQ),TQ)),(((TI,(TI,TI)),((UI,UI),UI)),((((((EO,EO),EO),(FO,(FO,FO))),((CM,CM),CM)),(WH,(WH,WH))),((YL,(YL,YL)),(ZL,(ZL,ZL))))))),(((MO,(MO,MO)),((NO,NO),NO)),(LK,(LK,LK))))),((((((((VD,VD),VD),(WD,(WD,WD))),((((VI,VI),VI),((WI,WI),WI)),(FH,(FH,FH)))),(((BG,BG),BG),((MH,(MH,MH)),(NH,(NH,NH))))),(((((ZH,(ZH,ZH)),((AI,AI),AI)),((HD,(HD,HD)),(((TK,(TK,TK)),((UK,UK),UK)),((LI,(LI,LI)),(MI,(MI,MI)))))),(((UC,UC),UC),(((UF,UF),UF),((VF,VF),VF)))),(((((RB,RB),RB),(((EC,EC),EC),((JF,(JF,JF)),(((TP,TP),TP),((UP,UP),UP))))),((((GH,GH),GH),((HH,HH),HH)),(((DD,DD),DD),(((MR,MR),MR),((ES,(ES,ES)),((FS,FS),FS)))))),(((BD,BD),BD),((ED,(ED,ED)),(((((KQ,KQ),KQ),(LQ,(LQ,LQ))),(BK,(BK,BK))),((CH,CH),CH))))))),((((BR,(BR,BR)),(CR,(CR,CR))),(BM,(BM,BM))),((GS,(GS,GS)),(HS,(HS,HS))))),((((GL,GL),GL),(HL,(HL,HL))),((((((((DR,DR),DR),((ER,ER),ER)),(IH,(IH,IH))),(CE,(CE,CE))),((RA,RA),RA)),(QA,(QA,QA))),((((KP,KP),KP),(LP,(LP,LP))),((NI,NI),NI)))))),(((((KE,(KE,KE)),((OJ,(OJ,OJ)),(((YM,YM),YM),((ZM,ZM),ZM)))),(((GO,(GO,GO)),(HO,(HO,HO))),((TN,TN),TN))),(((LE,LE),LE),((ME,ME),ME))),(((((KO,KO),KO),((LO,LO),LO)),(UM,(UM,UM))),(((LD,LD),LD),(MD,(MD,MD))))))),(((((ZC,(ZC,ZC)),(((IM,(IM,IM)),(JM,(JM,JM))),((TF,TF),TF))),(((((UH,UH),UH),((VH,VH),VH)),((BL,(BL,BL)),((CL,CL),CL))),(TB,(TB,TB)))),(((((ND,ND),ND),(OD,(OD,OD))),(KA,(KA,KA))),(((CT,CT),CT),(DT,(DT,DT))))),((((((FE,FE),FE),(((SG,SG),SG),((TG,TG),TG))),(((AS,AS),AS),((BS,BS),BS))),((((BA,(BA,BA)),(((((IE,(IE,IE)),((JE,JE),JE)),((YD,YD),YD)),((GC,GC),GC)),(((((CF,CF),CF),(((JK,JK),JK),((KK,KK),KK))),(((((MJ,MJ),MJ),(NJ,(NJ,NJ))),(NB,(NB,NB))),((MA,MA),MA))),((((VM,(VM,VM)),(WM,(WM,WM))),(((OP,OP),OP),((PP,PP),PP))),((RC,RC),RC))))),((CA,(CA,CA)),(DA,(DA,DA)))),(AA,(AA,AA)))),((((TD,TD),TD),((((ZP,ZP),ZP),(AQ,(AQ,AQ))),(IJ,(IJ,IJ)))),((((((((AN,(AN,AN)),(((HN,HN),HN),(IN,(IN,IN)))),((KM,KM),KM)),(((FJ,FJ),FJ),(GJ,(GJ,GJ)))),((OA,(OA,OA)),(PA,(PA,PA)))),((((((PM,PM),PM),(QM,(QM,QM))),((AB,(AB,AB)),(BB,(BB,BB)))),((QL,(QL,QL)),(RL,(RL,RL)))),((HA,HA),HA))),(((((NM,(NM,NM)),((OM,OM),OM)),(((((KL,KL),KL),(((TO,TO),TO),(UO,(UO,UO)))),(CK,(CK,CK))),((XF,XF),XF))),((OF,(OF,OF)),(((UG,UG),UG),(((DH,DH),DH),(EH,(EH,EH)))))),((JA,JA),JA))),(((((QT,QT),QT),(RT,(RT,RT))),((MQ,MQ),MQ)),((((XQ,XQ),XQ),((YQ,YQ),YQ)),((((VO,(VO,VO)),(WO,(WO,WO))),(AG,(AG,AG))),((((ZE,ZE),ZE),(AF,(AF,AF))),((UE,UE),UE)))))))))),((((((((EM,(EM,EM)),(FM,(FM,FM))),(AM,(AM,AM))),(TJ,(TJ,TJ))),(((EL,EL),EL),(FL,(FL,FL)))),(BF,(BF,BF))),(((((((EF,(EF,EF)),((((JI,JI),JI),(KI,(KI,KI))),(((XJ,XJ),XJ),(YJ,(YJ,YJ))))),((((JN,JN),JN),(KN,(KN,KN))),(JL,(JL,JL)))),((LM,(LM,LM)),((MM,MM),MM))),(((KD,KD),KD),(((CQ,(CQ,CQ)),((DQ,DQ),DQ)),((HI,HI),HI)))),(((((WE,WE),WE),(XE,(XE,XE))),((BE,BE),BE)),((ZD,ZD),ZD))),(((GA,GA),GA),(((JB,(JB,JB)),(((PE,PE),PE),((QE,QE),QE))),(ZA,(ZA,ZA)))))),((((((((QQ,QQ),QQ),(RQ,(RQ,RQ))),(((KG,(KG,KG)),(LG,(LG,LG))),((CG,CG),CG))),((((((PG,PG),PG),(QG,(QG,QG))),(((RO,RO),RO),((SO,SO),SO))),((TC,TC),TC)),((DB,DB),DB))),((((((BJ,BJ),BJ),(((UN,UN),UN),((VN,VN),VN))),(PD,(PD,PD))),(((CO,CO),CO),(DO,(DO,DO)))),((((AH,AH),AH),((BH,BH),BH)),(((PF,PF),PF),((QF,QF),QF))))),(((((MG,(MG,MG)),(NG,(NG,NG))),((YA,YA),YA)),((((((((PI,(PI,PI)),(((DP,DP),DP),(((NR,(NR,NR)),((OR,OR),OR)),((EQ,EQ),EQ)))),((NF,(NF,NF)),((HK,(HK,HK)),((IK,IK),IK)))),(IB,(IB,IB))),((((((TR,TR),TR),((UR,UR),UR)),((DK,(DK,DK)),((NN,(NN,NN)),(ON,(ON,ON))))),((((((US,US),US),((VS,VS),VS)),(ZR,(ZR,ZR))),(LH,(LH,LH))),((HE,HE),HE))),(((EI,EI),EI),((FI,FI),FI)))),(SA,(SA,SA))),(((DC,DC),DC),(((PH,PH),PH),((QH,QH),QH)))),(((AC,(AC,AC)),(((GI,GI),GI),((((YN,YN),YN),(ZN,(ZN,ZN))),((XK,XK),XK)))),((((DL,DL),DL),(((IO,IO),IO),((JO,JO),JO))),(XA,(XA,XA)))))),(((((VB,(VB,VB)),(((BC,BC),BC),((YF,(YF,YF)),((ZF,ZF),ZF)))),((((((AP,AP),AP),(((QS,QS),QS),((RS,RS),RS))),(YC,(YC,YC))),((DN,(DN,DN)),((EN,EN),EN))),(((YG,YG),YG),((AO,(AO,AO)),((BO,BO),BO))))),((((BP,BP),BP),(CP,(CP,CP))),((AD,AD),AD))),((((((IR,(IR,IR)),(JR,(JR,JR))),((YO,YO),YO)),(QO,(QO,QO))),((TE,TE),TE)),(((EA,(EA,EA)),(((((KC,KC),KC),((((FG,FG),FG),((RJ,(RJ,RJ)),((SJ,SJ),SJ))),(((GT,(GT,GT)),((HT,HT),HT)),((OK,OK),OK)))),((((LB,LB),LB),((MB,MB),MB)),(((YE,YE),YE),(((DJ,(DJ,DJ)),(EJ,(EJ,EJ))),(((GQ,GQ),GQ),(HQ,(HQ,HQ))))))),((HB,(HB,HB)),((JD,(JD,JD)),(((NQ,(NQ,NQ)),(OQ,(OQ,OQ))),((SF,SF),SF)))))),((HC,(HC,HC)),(((PN,PN),PN),(QN,(QN,QN))))))))),(((CD,(CD,CD)),((((ZI,ZI),ZI),(((EK,(EK,EK)),(FK,(FK,FK))),(AJ,(AJ,AJ)))),((EE,EE),EE))),(((HP,(HP,HP)),((IP,IP),IP)),(WL,(WL,WL))))))));
