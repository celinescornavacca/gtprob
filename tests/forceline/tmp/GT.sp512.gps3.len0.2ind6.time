182327.90user 51.92system 50:41:27elapsed 99%CPU (0avgtext+0avgdata 5218416maxresident)k
64inputs+8outputs (0major+8229560minor)pagefaults 0swaps
