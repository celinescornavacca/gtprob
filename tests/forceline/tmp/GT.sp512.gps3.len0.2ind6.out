-2424.81742427
